core/src/main/scala/org/locationtech/rasterframes/expressions/transformers/ReprojectGeometry.scala

```diff
@@ -34,15 +34,26 @@ import org.apache.spark.sql.types.DataType
 import org.apache.spark.sql.{Column, TypedColumn}
 import org.locationtech.rasterframes.encoders.CatalystSerializer
 import org.locationtech.rasterframes.jts.ReprojectionTransformer
+import org.locationtech.rasterframes.model.LazyCRS
 
-/**
-  *
-  *
-  * @since 11/29/18
-  */
+@ExpressionDescription(
+  usage = "_FUNC_(geom, srcCRS, dstCRS) - Reprojects the given `geom` from `srcCRS` to `dstCRS",
+  arguments = """
+  Arguments:
+    * geom - the geometry column to reproject
+    * srcCRS - the CRS of the `geom` column
+    * dstCRS - the CRS to project geometry into""",
+  examples = """
+  Examples:
+    > SELECT _FUNC_(geom, srcCRS, dstCRS);
+       ..."""
+)
 case class ReprojectGeometry(geometry: Expression, srcCRS: Expression, dstCRS: Expression) extends Expression
   with CodegenFallback with ExpectsInputTypes {
 
+  // TODO: Replace registration in `org.locationtech.rasterframes.functions.register`
```

Comment:
If it makes sense, let's hold off on merging this PR until we handle this TODO

Reply:
https://github.com/locationtech/rasterframes/issues/138